lib/js/_create/_build-new-package.js

```diff
@@ -36,6 +36,15 @@ function generateFolders(answers) {
 					if (err) {
 						console.error(err);
 					} else {
+						if (value === 'scss') {
+							config.CSSDirectoryStructure.forEach(subfolder => {
```

Comment:
we never check that `config.CSSDirectoryStructure` exists - see my other comment about changing the config structure